core-bundle/tests/Security/Authentication/Provider/AuthenticationProviderTest.php

```diff
@@ -292,7 +292,7 @@ public function testHandlesContaoUsers(): void
 
     public function testBeginsTwoFactorAuthenticationForContaoUsers(): void
     {
-        $user = $this->createPartialMock(BackendUser::class, ['save']);
+        $user = $this->createPartialMock(BackendUser::class, ['save', 'getUserIdentifier']);
```

Comment:
can you explain why we need to mock the method to return null instead of a property?

Reply:
Otherwise the tests fail.